Watchman.Discord/Program.cs

```diff
@@ -21,6 +22,8 @@ public static async Task Main(string[] args)
             
             var watchman = new WatchmanBot(configuration);
             var workflowBuilder = watchman.GetWorkflowBuilder();
+            workflowBuilder.ReactionAdded += (userMessage, socketMessageChannel, socketReaction) => ReactionsService.OnReactionAddedEvent(socketReaction);
+            workflowBuilder.ReactionRemoved += (userMessage, socketMessageChannel, socketReaction) => ReactionsService.OnReactionRemovedEvent(socketReaction);
```

Comment:
nie nie nie
w tej klasie jest tylko uruchomienie bota!
reakcje to zupełnie inna rzecz, nie powinna tutaj się znajdować